packages/core/src/com/environment-disconnected-error.ts

```diff
@@ -0,0 +1,6 @@
+export class EnvironmentDisconnectedError extends Error {
+    constructor(message: string) {
+        super(message);
+        this.name = 'EnvironmentDisconnectedError';
```

Comment:
use new.target? https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-2.html#support-for-newtarget

Reply:
I don't need this here. I don't have any extra prototype chain